Sources/BokenEngine/Classes/StoryParser.swift

```diff
@@ -53,7 +69,9 @@ class StoryParser {
     }
 
     func decodeJSON(resourceName: String) throws -> AppDescription? {
-        let contents = try readLocalFile(resourceName: resourceName)
+        let contents = isRunningTest() ?
```

Comment:
I think this condition `isRunningTest()` should be outside of the class. I think it should be part of the app bootstrapping somewhere else. But we can discuss it later since that change has nothing to do with this PR.

Reply:
Ok. Will leave it as is for the moment.